test/tests/performance/pbs_preemptperformance.py

```diff
@@ -495,6 +495,7 @@ def test_preemption_with_unrelated_soft_limits(self):
             j = Job(TEST_USER2, attrs=a)
             j.set_sleep_time(3000)
             self.server.submit(j)
+        time.sleep(15)
```

Comment:
Put a self.logger.info() ahead of this sleep saying the reason why we're sleeping.

Reply:
Good catch! Actually, there was no reason to sleep here. I added that because I was trying to run gprof compiled scheduler and I used to swap the normal sched binary with gprof compiled sched binary in these 15 seconds :)
I did not clean up well